lib/index.js

```diff
@@ -4,6 +4,19 @@
 const mailjet = require('node-mailjet');
 const { removeUndefined } = require('strapi-utils');
 
+const normalizeTo = to => {
+  return Array.isArray(to) ?
+   to.map(i =>
+     typeof i === 'string'
+      ? {Email: i}
+      : i) : [
+        {
+          Email: to || settings.defaultTo,
+          Name: toName || settings.defaultToName,
+        },
```

Comment:
Looks like we're currently not passing in the `toName` and `settings` to `normalizeTo()` - could you please add and ensure it works? :)